### PR TITLE
Update outdated files in the dev-1.20-ko.2 (p2)

### LINKED ISSUE #25911: Outdated files in the dev-1.20-ko.2 branch
# This is a Bug Report
## Problem
Outdated files in the upstream/dev-1.20-ko.2 branch.

### 27 files to be modified 
1. [ ] M1.  `content/en/community/static/cncf-code-of-conduct.md`  | 2(+XS) 2(-)
1. [ ] M2.  `content/en/docs/concepts/cluster-administration/kubelet-garbage-collection.md`  | 2(+XS) 2(-)
1. [ ] M3.  `content/en/docs/concepts/cluster-administration/networking.md`  | 1(+XS) 1(-)
1. [ ] M4.  `content/en/docs/concepts/cluster-administration/system-metrics.md`  | 23(+S) 21(-)
1. [ ] M5.  `content/en/docs/concepts/configuration/configmap.md`  | 1(+XS) 1(-)
1. [ ] M6.  `content/en/docs/concepts/configuration/manage-resources-containers.md`  | 1(+XS) 1(-)
1. [ ] M7.  `content/en/docs/concepts/configuration/secret.md`  | 8(+XS) 1(-)
1. [ ] M8.  `content/en/docs/concepts/extend-kubernetes/operator.md`  | 1(+XS) 1(-)
1. [ ] M9.  `content/en/docs/concepts/overview/kubernetes-api.md`  | 1(+XS) 1(-)
1. [ ] M10.  `content/en/docs/concepts/overview/working-with-objects/common-labels.md`  | 3(+XS) 3(-)
1. [ ] M11.  `content/en/docs/concepts/policy/pod-security-policy.md`  | 21(+S) 6(-)
1. [ ] M12.  `content/en/docs/concepts/storage/persistent-volumes.md`  | 1(+XS) 1(-)
1. [ ] M13.  `content/en/docs/concepts/workloads/_index.md`  | 54(+M) 30(-)
1. [ ] M14.  `content/en/docs/concepts/workloads/controllers/cron-jobs.md`  | 31(+M) 0(-)
1. [ ] M15.  `content/en/docs/concepts/workloads/controllers/job.md`  | 7(+XS) 3(-)
1. [ ] M16.  `content/en/docs/concepts/workloads/pods/_index.md`  | 29(+S) 0(-)
1. [ ] M17.  `content/en/docs/concepts/workloads/pods/init-containers.md`  | 2(+XS) 2(-)
1. [ ] M18.  `content/en/docs/concepts/workloads/pods/pod-topology-spread-constraints.md`  | 1(+XS) 1(-)
1. [ ] M19.  `content/en/docs/reference/_index.md`  | 1(+XS) 2(-)
1. [ ] M20.  `content/en/docs/reference/using-api/client-libraries.md`  | 1(+XS) 0(-)
1. [ ] M21.  `content/en/docs/setup/production-environment/container-runtimes.md`  | 18(+S) 0(-)
1. [ ] M22.  `content/en/docs/tasks/access-application-cluster/port-forward-access-application-cluster.md`  | 24(+S) 3(-)
1. [ ] M23.  `content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md`  | 7(+XS) 7(-)
1. [ ] M24.  `content/en/docs/tasks/debug-application-cluster/debug-pod-replication-controller.md`  | 1(+XS) 1(-)
1. [ ] M25.  `content/en/docs/tasks/manage-hugepages/scheduling-hugepages.md`  | 10(+S) 8(-)
1. [ ] M26.  `content/en/docs/tasks/manage-kubernetes-objects/kustomization.md`  | 1(+XS) 1(-)
1. [ ] M27.  `content/en/docs/tutorials/stateless-application/expose-external-ip-address.md`  | 82(+M) 80(-)

## Proposed Solution

Use `git diff` to check what is changed in the upstream. And apply the upstream changes manually 
to the `content/ko` of `upstream/dev-1.20-ko.2` branch.

For example:
```
# checkout `upstream/dev-1.20-ko.2`
...
# check what is updated in the upstream 
git diff upstream/dev-1.20-ko.1 upstream/dev-1.20-ko.2 -- content/en/community/static/cncf-code-of-conduct.md
# apply changes to content/ko
vi content/ko/community/static/cncf-code-of-conduct.md
...
# commit and push
...
# make PR to `upstream/dev-1.20-ko.2`
```

## Pages to Update

Pages in content/ko

### PR DESCRIPTION
**Pull request by seokho-son**
Partially fix #25911 (Part 2: M17 ~ M27).

Update the following outdated files in the dev-1.20-ko.2 branch.

1. [x] M17.  `content/en/docs/concepts/workloads/pods/init-containers.md`  | 2(+XS) 2(-)
1. [x] M18.  `content/en/docs/concepts/workloads/pods/pod-topology-spread-constraints.md`  | 1(+XS) 1(-)
1. [x] M19.  `content/en/docs/reference/_index.md`  | 1(+XS) 2(-)
1. [x] M20.  `content/en/docs/reference/using-api/client-libraries.md`  | 1(+XS) 0(-)
1. [x] M21.  `content/en/docs/setup/production-environment/container-runtimes.md`  | 18(+S) 0(-)
1. [x] M22.  `content/en/docs/tasks/access-application-cluster/port-forward-access-application-cluster.md`  | 24(+S) 3(-)
1. [x] M23.  `content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md`  | 7(+XS) 7(-)
1. [x] M24.  `content/en/docs/tasks/debug-application-cluster/debug-pod-replication-controller.md`  | 1(+XS) 1(-)
1. [x] M25.  `content/en/docs/tasks/manage-hugepages/scheduling-hugepages.md`  | 10(+S) 8(-)
1. [x] M26.  `content/en/docs/tasks/manage-kubernetes-objects/kustomization.md`  | 1(+XS) 1(-)
1. [x] M27.  `content/en/docs/tutorials/stateless-application/expose-external-ip-address.md`  | 82(+M) 80(-)

### CODE REVIEW

File: content/ko/docs/tasks/access-application-cluster/port-forward-access-application-cluster.md [hunk 1]
```diff
@@ -157,10 +158,16 @@ min-kubernetes-server-version: v1.10
...
     ```
-    I0710 14:43:38.274550    3655 portforward.go:225] Forwarding from 127.0.0.1:7000 -> 6379
-    I0710 14:43:38.274797    3655 portforward.go:225] Forwarding from [::1]:7000 -> 6379
+    Forwarding from 127.0.0.1:7000 -> 6379
+    Forwarding from [::1]:7000 -> 6379
     ```
 
+{{< note >}}
+
+`kubectl port-forward` 은 리턴하지 않으므로, 이 연습을 계속하려면 다른 터미널을 열어야 한다.
```
Comment (pjhwa) [inline content/ko/docs/tasks/access-application-cluster/port-forward-access-application-cluster.md:167]:
```suggestion
`kubectl port-forward` 는 프롬프트를 리턴하지 않으므로, 이 연습을 계속하려면 다른 터미널을 열어야 한다.
```

- 의역을 해봤습니다.

File: content/ko/docs/tasks/access-application-cluster/port-forward-access-application-cluster.md [hunk 2]
```diff
@@ -179,7 +186,23 @@ min-kubernetes-server-version: v1.10
     PONG
     ```
 
+### 선택적으로 _kubectl_ 이 로컬 포트를 선택하게 하기 {#let-kubectl-choose-local-port}
+
+만약 특정 로컬 포트가 필요하지 않다면, `kubectl` 이 로컬 포트를 선택 및 할당하게 하여,
+조금 더 단순한 문법으로 로컬 포트 충돌 관리를 위한
+부담을 경감할 수 있다.
```
Comment (pjhwa) [inline content/ko/docs/tasks/access-application-cluster/port-forward-access-application-cluster.md:193]:
```suggestion
부담을 줄일 수 있다.
```

- 좀 더 자연스럽게 수정해봤습니다.

File: content/ko/docs/tasks/manage-hugepages/scheduling-hugepages.md [hunk 3]
```diff
@@ -102,17 +104,18 @@ spec:
...
-`HugePageStorageMediumSize` [기능
-게이트](/ko/docs/reference/command-line-tools-reference/feature-gates/)를 사용하여 비활성화할 수 있다.
+  다른 컴퓨트 리소스와 비슷한 리소스쿼터(ResourceQuota)를 통해 제어할 수
+  있다.
+- 다양한 크기의 huge page 지원이 기능 게이트로 제공된다.
+  {{<glossary_tooltip text="kubelet" term_id="kubelet" >}} 및
+  {{<glossary_tooltip text="kube-apiserver" term_id="kube-apiserver" >}}
+  (`--feature-gates=HugePageStorageMediumSize=true`)의 `HugePageStorageMediumSize`
+  [기능 게이트](/ko/docs/reference/command-line-tools-reference/feature-gates/)
+  를 사용하여 비활성화할 수 있다.
```
Comment (pjhwa) [inline content/ko/docs/tasks/manage-hugepages/scheduling-hugepages.md:121]:
```suggestion
  [기능 게이트](/ko/docs/reference/command-line-tools-reference/feature-gates/)를
  사용하여 비활성화할 수 있다.
```

File: content/ko/docs/tasks/manage-kubernetes-objects/kustomization.md [hunk 4]
```diff
@@ -47,8 +47,7 @@ Kustomize는 쿠버네티스 구성을 사용자 정의화하는 도구이다. 
 
 ### 리소스 생성
 
-컨피그 맵과 시크릿은 파드같은 다른 쿠버네티스 오브젝트에서 사용되는 설정이나 민감한 데이터를 가지고 있다.
-컨피그 맵이나 시크릿의 실질적인 소스는 일반적으로 `.properties` 파일이나 ssh key 파일과 같은 것들은 클러스터 외부에 있다.
+컨피그 맵과 시크릿은 파드같은 다른 쿠버네티스 오브젝트에서 사용되는 설정이나 민감한 데이터를 가지고 있다. 컨피그 맵이나 시크릿의 실질적인 소스는 일반적으로 `.properties` 파일이나 ssh key 파일과 같은 것들은 클러스터 외부에 있다.
```
Comment (pjhwa) [inline content/ko/docs/tasks/manage-kubernetes-objects/kustomization.md:50]:
```suggestion
컨피그 맵과 시크릿은 파드와 같은 다른 쿠버네티스 오브젝트에서 사용되는 설정이나 민감한 데이터를 가지고 있다. 컨피그 맵이나 시크릿의 실질적인 소스는 일반적으로 `.properties` 파일이나 ssh key 파일과 같은 것들은 클러스터 외부에 있다.
```

**Review by pjhwa — COMMENTED** [review]
@seokho-son 감사합니다!

몇 가지 작은 의견 드립니다.

**Comment by seokho-son** [comment]
@pjhwa 리뷰 감사합니다. 의견 모두 반영하여 스쿼시하였습니다 ~ !

**Comment by pjhwa** [comment]
수고하셨습니다! 👍 

/lgtm

**Comment by seokho-son** [comment]
PTAL: @kubernetes/sig-docs-ko-owners

**Pull request merged**